yellowbrick/text/postag.py

```diff
@@ -20,125 +20,371 @@
+        colors = resolve_colors(
+            n_colors=len(self.tag_map), 
+            colormap=self.colormap, 
+            colors=self.colors
+        )
+        self.ax.bar(
+            range(len(self.tag_map)), 
+            list(self.tag_map.values()), 
+            color=colors
+        )
+        return self.ax
+        
+    def finalize(self, **kwargs):
+        """
+        Finalize the plot with ticks, labels, and title
 
-        Text documents must be tokenized and tagged before passing to fit()
+        Parameters
+        ----------
+        kwargs: dict
+            generic keyword arguments.
         """
-        self.tagged = [
-            (self.TAGS.get(tag),tok) for tok, tag in tagged_tuples
-        ]
-        #
-        # print(' '.join((colorize(token, color) for color, token in self.tagged)))
-        # print('\n')
+        # NOTE: not deduping here, so this is total, not unique
+        self.set_title(
+            "PosTag plot for {}-token corpus".format(
+                (sum(self.tag_map.values()))
+            )
+        )
+        
+        self.ax.set_xticks(range(len(self.tag_map)))
+        self.ax.set_xticklabels(list(self.tag_map.keys()), rotation=90)
+        
+        # Set the axis labels
+        self.ax.set_xlabel(
+            '{} part-of-speech tags'.format(
+                self.tagset_names[self.tagset]
+            )
+        )
+        self.ax.set_ylabel('Count')
+
+##########################################################################
+## Quick Method
+##########################################################################
+
+def postag(X, y=None, ax=None, tagset="penn_treebank", colormap=None, 
+            colors=None, **kwargs):
+    """
+    Display a barchart with the counts of different parts of speech
+    in X, which consists of a part-of-speech-tagged corpus, which the
+    visualizer expects to be a list of lists of lists of (token, tag) 
+    tuples.
+
+    Parameters
+    ----------
+    ax : matplotlib axes
+        The axes to plot the figure on.
+    tagset: string
+        The tagset that was used to perform part-of-speech tagging.
+        Either "penn_treebank" or "universal", defaults to "penn_treebank".
+        Use "universal" if corpus has been tagged using SpaCy.
+    colors : list or tuple of colors
+        Specify the colors for each individual part-of-speech.
+    colormap : string or matplotlib cmap
+        Specify a colormap to color the parts-of-speech.
+    kwargs : dict
+        Pass any additional keyword arguments to the PosTagVisualizer. 
+
+    Returns
+    -------
+    ax : matplotlib axes
+        Returns the axes on which the PosTagVisualizer was drawn.
+    """
+    # Instantiate the visualizer
+    visualizer = PosTagVisualizer(
+        ax, tagset, colors, colormap, **kwargs
+    )
+
+    # Fit and transform the visualizer (calls draw)
+    visualizer.fit(X, y, **kwargs)
+
+    # Return the axes object on the visualizer
+    return visualizer.ax
```

Comment:
Part of #600 is to refactor quick methods to return the visualizer instead of the ax - not a big deal but if you want to get ahead of the game, I'd appreciate it!

Reply:
Got it; fixed.